terraform/cloudsql/postgresql/bind/variables.tf

```diff
@@ -2,7 +2,10 @@ variable "db_name" { type = string }
 variable "hostname" { type = string }
 variable "admin_username" { type = string }
 variable "admin_password" { type = string }
-variable "use_tls" { type = bool }
+#variable "use_tls" { type = bool }
```

Comment:
I'd rather remove this variable, rather than comment it out. Just leads me to search for context as to why it's there...

Reply:
In general I agree, but in this case I was following a convention:
- https://github.com/cloudfoundry/csb-brokerpak-gcp/blob/9f924c5b66b276dba46295e769e49569296905af/google-postgresql.yml#L52-L56
- https://github.com/cloudfoundry/csb-brokerpak-gcp/blob/9f924c5b66b276dba46295e769e49569296905af/terraform/cloudsql/postgresql/provision/variables.tf#L20

Given the next story is to make this optional (and hence uncomment the variable), I'm not sure it's worth questioning the convention.